accounts/keystore/keystore_passphrase.go

```diff
@@ -97,7 +102,26 @@ func (ks keyStorePassphrase) StoreKey(filename string, key Key, auth string) err
 	if err != nil {
 		return err
 	}
-	return writeKeyFile(filename, keyjson)
+	// Write into temporary file
+	tmpName, err := writeTemporaryKeyFile(filename, keyjson)
+	if err != nil {
+		return err
+	}
+	if ks.skipKeyFileVerification {
+	        return os.Rename(tmpName, filename)
+	}
+	// Verify that we can decrypt the file with the given password.
+	_, err = ks.GetKey(key.GetAddress(), tmpName, auth)
+	if err != nil {
+		msg := "An error was encountered when saving and verifying the keystore file. \n" +
+			"This indicates that the keystore is corrupted. \n" +
+			"The corrupted file is stored at \n%v\n" +
+			"Please file a ticket at:\n\n" +
+			"https://github.com/klaytn/klaytn/issues" +
+			"The error was : %s"
```

Comment:
```suggestion
			"The error was : %w"
```
Below link would be helpful.
https://blog.golang.org/go1.13-errors

Reply:
The suggested change has been applied at https://github.com/klaytn/klaytn/pull/725/commits/d3ac32bc991fc06d1c92b7dd70bc0ba8e31bbcbb